spec/podfile/target_definition_spec.rb

```diff
@@ -571,6 +571,68 @@ module Pod
         @parent.swift_version.should == '2.3'
       end
 
+      it 'stores swift version requirements as strings' do
+        @parent.store_swift_version_requirements('>= 3.0', '< 4.0')
+        @parent.send(:get_hash_value, 'swift_version_requirements').should == [
+          '>= 3.0',
+          '< 4.0',
+        ]
+      end
+
+      it 'stores swift version requirements as versions' do
+        @parent.store_swift_version_requirements(Version.new('3.0'), Version.new('4.0'))
+        @parent.send(:get_hash_value, 'swift_version_requirements').should == [
+          '3.0',
+          '4.0',
+        ]
+      end
```

Comment:
maybe add another test to make sure that when passing a single argument it gets converted to an array correctly

```ruby
@parent.store_swift_version_requirements('3.0')
@parent.send(:get_hash_value, 'swift_version_requirements').should == [
    '3.0',
 ]
```

I know it's covered at the `swift_versions` level though

Reply:
done